### PR TITLE
Crs 729  password for docs

### PR DESCRIPTION
**Pull request by daniel-packard**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -32,6 +32,43 @@ const formatString = (str) => {
...
+  let layout = document.querySelector(".layout");
+  layout.classList.add("blurred");
+  window.scrollTo(0,0);
+}
+
+function unlock(e) {
+    e.preventDefault();
+    var password = document.getElementById("password").value;
+
+    if(hash(password) === 1032675042) {
```
Comment (daniel-packard) [inline src/pages/index.js:61]: not super secure, if someone wants to log in and look at the javascript... but at least it's hashed..

just trying to keep out the riff-raff

File: src/styles/layout.scss [hunk 2]
```diff
@@ -482,4 +483,116 @@ body {
     color: $brand-dark;
     text-align: center;
     font-size: small;
+}
+
+.lockout-overlay {
```
Comment (ghost) [inline src/styles/layout.scss:488]: you can remove one .lockout-overlay right?

Comment (daniel-packard) [reply src/styles/layout.scss:488]: Absolutely! good catch

File: src/pages/index.js [hunk 3]
```diff
@@ -54,20 +87,17 @@ if (typeof window !== 'undefined') {
             return `<div class="${c}"><a href="#${link}">${item.value}</a></div>`;
         }).join("");
 
-        console.log(markup);
         tocDiv.innerHTML = '<h1 className="table-heading">Table of Contents</h1>' + markup;
     }
 
     var toc = {}
```
Comment (ghost) [inline src/pages/index.js]: I think we don't use toc anymore. It's my bad that I didn't remove it while I was working on it! Can you please remove toc and may be rename toc2 to toc?

Comment (ghost) [reply src/pages/index.js]: Besides this, everything else looks good!

**Pull request merged**